lib/node_modules/@stdlib/math/base/special/pow/src/main.c

```diff
@@ -0,0 +1,725 @@
+
+// 0x41e00000 = 1105199104 => 0 10000011110 00000000000000000000 => biased exponent: 1054 = 31+1023 => 2^31
+static const uint32_t HIGH_BIASED_EXP_31 = 0x41e00000;
+
+// 0x43f00000 = 1139802112 => 0 10000111111 00000000000000000000 => biased exponent: 1087 = 64+1023 => 2^64
+static const uint32_t HIGH_BIASED_EXP_64 = 0x43f00000;
+
+// 0x40900000 = 1083179008 => 0 10000001001 00000000000000000000 => biased exponent: 1033 = 10+1023 => 2^10 = 1024
+static const int32_t HIGH_BIASED_EXP_10 = 0x40900000;
+
+// 0x4090cc00 = 1083231232 => 0 10000001001 00001100110000000000
+static const int32_t HIGH_1075 = 0x4090cc00;
+
+// 0xc090cc00 = 3230714880 => 1 10000001001 00001100110000000000
+static const int32_t HIGH_NEG_1075 = 0xc090cc00;
+
+static const uint32_t HIGH_NUM_NONSIGN_BITS = 31;
+
+// -(1024-log2(ovfl+.5ulp))
+static const double OVT = 8.0085662595372944372e-17;
+
+// Log workspace:
+double LOG_WORKSPACE[2] = { 0.0, 0.0 };
+
+/* Begin auto-generated functions. The following functions are auto-generated. Do not edit directly. */
+
+// BEGIN: polyval_l
+
+/**
+* Evaluates a polynomial.
+*
+* ## Notes
+*
+* -   The implementation uses [Horner's rule][horners-method] for efficient computation.
+*
+* [horners-method]: https://en.wikipedia.org/wiki/Horner%27s_method
+*
+* @param x    value at which to evaluate the polynomial
+* @return     evaluated polynomial
+*/
+static double polyval_l( const double x ) {
+	return 0.5999999999999946 + (x * (0.4285714285785502 + (x * (0.33333332981837743 + (x * (0.272728123808534 + (x * (0.23066074577556175 + (x * 0.20697501780033842)))))))));
+}
+
+// END: polyval_l
+
+// BEGIN: polyval_w
+
+/**
+* Evaluates a polynomial.
+*
+* ## Notes
+*
+* -   The implementation uses [Horner's rule][horners-method] for efficient computation.
+*
+* [horners-method]: https://en.wikipedia.org/wiki/Horner%27s_method
+*
+* @param x    value at which to evaluate the polynomial
+* @return     evaluated polynomial
+*/
+static double polyval_w( const double x ) {
+	return 0.5 + (x * (-0.3333333333333333 + (x * 0.25)));
+}
+
+// END: polyval_w
+
+// BEGIN: polyval_p
+
+/**
+* Evaluates a polynomial.
+*
+* ## Notes
+*
+* -   The implementation uses [Horner's rule][horners-method] for efficient computation.
+*
+* [horners-method]: https://en.wikipedia.org/wiki/Horner%27s_method
+*
+* @param x    value at which to evaluate the polynomial
+* @return     evaluated polynomial
+*/
+static double polyval_p( const double x ) {
+	return 0.16666666666666602 + (x * (-0.0027777777777015593 + (x * (0.00006613756321437934 + (x * (-0.0000016533902205465252 + (x * 4.1381367970572385e-8)))))));
+}
+
+// END: polyval_p
+
+/* End auto-generated functions. */
+
+/**
+* Evaluates the exponential function when \\( y = \pm \infty\\).
+*
+* @param x    input value
+* @param y    input value
+* @return	  output value
+*
+* @example
+* double out = y_is_infinite( -1.0, Infinity );
+* // returns NaN
+*
+* @example
+* double out = y_is_infinite( 1.5, -Infinity );
```

Comment:
```suggestion
* double out = y_is_infinite( -1.0, 1.0/0.0 );
* // returns NaN
*
* @example
* double out = y_is_infinite( 1.5, -1.0/0.0 );
```

`Infinity` is a JavaScript global, not a C global.